INSTALL.md

```diff
@@ -15,7 +15,7 @@
 | OpenCV       | >= 2.4.9    | core, highgui, imgproc                            |
 | OpenAL Soft  | >= 1.16.0   |                                                   |
 | filter_audio | most recent |                                                   |
-
+| qrcode       | most recent |                                                   |
```

Comment:
https://pypi.python.org/pypi/qrcode ?

Also, why would you put a requirement on most recent? Does your code use some functionality that is only available in most recent version?